src/Files.App/Views/LayoutModes/ColumnViewBase.xaml

```diff
@@ -284,7 +284,7 @@
 										Visibility="{x:Bind IsShortcutItem}">
 										<FontIcon
 											FontFamily="{StaticResource CustomGlyph}"
-											FontSize="14"
+											FontSize="9"
```

Comment:
Value should be evenly divisible by 4 so let's try with 12.

Reply:
I'll put all the multiple of four so you can choose the one you prefer. I think 12px may be too big. 8px would be ideal.

Original :
![original](https://user-images.githubusercontent.com/110472580/191914948-22ab1ae9-2d14-4780-8fc1-28e6860878db.png)

12px:
![12px](https://user-images.githubusercontent.com/110472580/191914966-6b14f98d-b13e-43b8-ab07-29e8a83167a6.png)

8px:
![8px](https://user-images.githubusercontent.com/110472580/191914984-6c073632-54ce-4865-9101-cc990a50476a.png)

4px:
![4px](https://user-images.githubusercontent.com/110472580/191915038-b31028f7-7150-4413-9697-e46a7db67fac.png)


Also my bad for the "not approved issue" part. Did not saw the "triage approved" tag being removed in it.

Reply:
Let's go with `8`.
The only reason I locked the issue was because we like to figure out the details beforehand, otherwise we end up with long discussions on the pull request and it's easier to come up with the requirements before the work is done.

Reply:
Done.